.github/CONTRIBUTING.md

```diff
@@ -25,4 +25,5 @@
 2. Fork the repository on GitHub.
 3. Check the [Contribution Guidelines](https://github.com/mrdoob/three.js/wiki/How-to-contribute-to-three.js).
 4. Make changes to your clone of the repository.
-5. Submit a pull request. Don't include build files in the PR.
+5. If your changes leads to a change in examples, make a new one screenshot with `npx FILE=<name> npm run make-screenshot`.
```

Comment:
What's npx? 🤔

Reply:
Yep, `npx cross-env FILE=<name> npm run make-screenshot`, npx it's a wrapper that run a command like it will be in package.json. We need it because we need cross-platform solution with environment variables.

Reply:
Is it supposed to be `npx FILE=<example_filename> npm run make-screenshot`?

Reply:
Filename without extension. With `npx cross-env EXAMPLE=<name> npm run make-screenshot` probably more clear to everyone?

Reply:
Why without the extension? This would't work?

```shell
npx FILE=webgl_geometry_cube.html npm run make-screenshot
```

Reply:
I will create this modification tomorrow, thanks for review. By the way right now Travis configured in `ignore fails` mode and will not complain about error in diff test. When we will be sure in it, we can remove `ignore fails` mode. Also Travis not supported color terminal, probably we need to write your own image console printer or switch to CircleCI, they [support](https://circleci.com/gh/munrocket/puppeteer-three/157#tests/containers/3) colors.